### PR TITLE
feat: Add a high-level page about the dev toolbar

### PR DESCRIPTION
**Pull request by Princesseuh**
#### Description (required)

Right now it's not super documented that the dev toolbar *exists*, despite being at the bottom of every single page 😅

This page hopes to serve as an overview of the dev toolbar and its features.

### CODE REVIEW

**Comment by sarah11918** [comment]
Thank you! Will take a look at this in the next couple of days, when I can sit down and get comfy with it!  :raised_hands:

**Comment by mingjunlu** [comment]
This is awesome! The dev toolbar definitely needs more attention 🙌

File: src/content/docs/en/guides/dev-toolbar.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+### Settings
+
+The Settings app allows you to toggle different settings for the development toolbar, such as verbose logging, and the ability to disable notifications.
+
+## Extending the dev toolbar
+
+[Astro integrations](/en/guides/integrations-guide/) can add new apps to the dev toolbar, allowing you to extend it with custom tools that are specific to your project. To see a list of integrations that provide dev toolbar apps, see the "Dev Toolbar" section of the [integrations list](https://astro.build/integrations/?search=&categories%5B%5D=toolbar).
+
+```shell
```
Comment (mingjunlu) [inline src/content/docs/en/guides/dev-toolbar.mdx:39]: Is this accidentally added? 👀

File: src/content/docs/en/guides/dev-toolbar.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Astro
+
+The Astro app provides easy access to different information about the current project and links about Astro. Notably, it provides a one-click access to the Astro documentation, GitHub repository, and Discord server.
+
+This app also include a "Copy debug info" button, which will copy the output of the [`astro info`](/en/reference/cli-reference/#astro-info) command to your clipboard. This can be useful when asking for help or reporting issues.
+
+### X-ray
+
+The X-ray app allows you to inspect the [Islands](/en/concepts/islands/) on the current page, showing you the properties that are being passed to each islands, and the client directive that is being used to render them.
```
Comment (mingjunlu) [inline src/content/docs/en/guides/dev-toolbar.mdx:19]:
```suggestion
The X-ray app allows you to inspect the [Islands](/en/concepts/islands/) on the current page, showing you the properties that are being passed to each island, and the client directive that is being used to render them.
```

File: src/content/docs/en/guides/dev-toolbar.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+### Per-user
+
+If you want to disable the dev toolbar for yourself on a specific project, you can do so using the `astro preferences` command.
+
+```shell
+astro preferences disable devToolbar
+```
+
+To disable the devToolbar in all Astro projects on the current machine:
```
Comment (mingjunlu) [inline src/content/docs/en/guides/dev-toolbar.mdx:65]: nit: for word consistency

```suggestion
To disable the dev toolbar in all Astro projects on the current machine:
```

File: src/content/docs/en/guides/dev-toolbar.mdx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+```shell
+astro preferences disable --global devToolbar
+```
+
+The devToolbar can later be enabled with:
```
Comment (mingjunlu) [inline src/content/docs/en/guides/dev-toolbar.mdx:71]: nit: for word consistency

```suggestion
The dev toolbar can later be enabled with:
```

**Review by mingjunlu — COMMENTED** [review]
Well-written! I find it easy to read and understand. Leaving a couple suggestions here

File: src/content/docs/en/guides/dev-toolbar.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Extending the dev toolbar
+
+[Astro integrations](/en/guides/integrations-guide/) can add new apps to the dev toolbar, allowing you to extend it with custom tools that are specific to your project. To see a list of integrations that provide dev toolbar apps, see the "Dev Toolbar" section of the [integrations list](https://astro.build/integrations/?search=&categories%5B%5D=toolbar).
+
+## Disabling the dev toolbar
+
+### Per-project
+
+To disable the dev toolbar for everyone working on a project, you can set `devToolbar: false` in your project's [config file](/en/reference/config-reference/).
```
Comment (mingjunlu) [inline src/content/docs/en/guides/dev-toolbar.mdx:43]: Found a broken link here

```suggestion
To disable the dev toolbar for everyone working on a project, you can set `devToolbar: false` in your project's [config file](/en/reference/configuration-reference/).
```

**Review by SatanshuMishra — COMMENTED** [review]
This is a great addition! Will definitly give people an insight into how to use the development toolbar. The page was easy to ready and understand. Amazing work making this!

**Comment by sarah11918** [comment]
@Princesseuh Would love for you to review the changes I made here! Then, when you're happy, I think this is good to merge!

**Comment by Princesseuh** [comment]
> @Princesseuh Would love for you to review the changes I made here! Then, when you're happy, I think this is good to merge!

Looks great to me! GitHub should really add a way to review my own PR

**Comment by sarah11918** [comment]
!coauthor

**Pull request merged**